app/assets/stylesheets/limber/plate.scss

```diff
@@ -46,12 +46,16 @@
       display: inline-block;
       border: 2px $gray-800 solid;
       border-radius: 7px;
-      font-size: small;
 
       .tag {
         display: none;
       }
 
+      // As the tag index gets longer, scale back the font size
+      .tag-3 {
+        font-size: 80%;
+      }
+
```

Comment:
For longer tag indicies, we scale the font. I did try some of CSS inbuilt scaling options, but wasn't successful.